src/commands/get-pr-info.yml

```
@@ -45,15 +45,15 @@ steps:
 
         PR_TITLE=$(echo $PR_RESPONSE | jq -e '.title' | tr -d '"')
```

It seems double quotes are already being trimmed here - maybe it's worth just updating these to trim off single quotes too?
```suggestion
        PR_TITLE=$(echo $PR_RESPONSE | jq -e '.title' | tr -d "\"'")
```

---

+1 on this! I think it'll be cleaner to have all the string cleaning in one place :)